CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 This changelog follows the format defined here: https://keepachangelog.com/en/1.0.0/
 
+## [1.7.0] - 2019-10-12
+This release primarily updates several FireMarshal defaults to recent
+upstreamed versions, notably: Linux (v5.3), Fedora (20190126), and Buildroot (2019.08).
+
+Also, offical documentation has moved to: https://firemarshal.readthedocs.io/en/latest/
+
+### Added
+* PR #47 enables custom arguments to spike and qemu (see also PR #48 that adds an example RoCC accelerator workload that uses these arguments)
+* PR #61 includes an example sha3 workload (matches example RTL in chipyard)
+
+### Changed
+* PR #40 Enables alternative package manager mirrors in Fedora (if the default mirrors go down)
+* PR #41 Switches the default Fedora image from a nightly build to a stable release.
+* PR #44 Enables compressed riscv instructions (RVC) by default in Linux
```

Comment:
I wasn't paying too close attention but didn't you effectively revert this?

Reply:
I thought I had, but it is still in the configs. I'm not sure why that was causing a merge conflict. The version in master definitely has it enabled now though.